packages/pluggableWidgets/video-player-native/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
```

Comment:
do we need to change the widget version?

Reply:
I think so, because we updated a package and it should be a new version

Reply:
it is not updated in the PR

Reply:
you've already updated the version to 5.0.0 for the video player, but we haven't released that version yet

Reply:
You can see here: https://github.com/mendix/native-widgets/pull/97/files#diff-fe6ff758cd50fbf3912828a6e368d1e1a1517fd7393be5f87ed3d2d3b49b5458